### PR TITLE
[POM-205] Add Statements static page

### PR DESCRIPTION
**Pull request by neu5**
- routing from filter sites needs to be added

### CODE REVIEW

File: src/app/statement/statement.component.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  accordionList: Array<Statement>;
+  shouldScroll: boolean;
+
+  @ViewChild('accordion') accordion!: MatAccordion;
+
+  constructor(private translateService: TranslateService) {
+    this.shouldScroll = false; // TODO: should be retrieved from the passed parameters
+    const shouldBeExpanded = !this.shouldScroll;
+
+    this.accordionList = [
```
Comment (broslukasz) [inline src/app/statement/statement.component.ts:30]: no business logic ina a constructor. it is enough to have it assign to component preperty.

File: src/app/statement/statement.component.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      {
+        id: 'panel-1',
+        title: this.translateService.instant('INFORMATOR_MOST_IMPORTANT_HEADER'),
```
Comment (broslukasz) [inline src/app/statement/statement.component.ts:33]: it looks is will not be handled after language change

File: src/app/statement/statement.component.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <header class="text-center">
+    <p class="font-small font-yellow fw-bold mb-1">{{ "HELPUA" | translate }}</p>
+    <h1 class="header-2 font-darkblue fw-bold">{{ "STATEMENT" | translate }}</h1>
+  </header>
+
+  <mat-accordion #accordion="matAccordion" multi>
+    <mat-expansion-panel *ngFor="let panel of accordionList" [expanded]="panel.isExpanded">
+      <mat-expansion-panel-header>
+        <mat-panel-title>
+          {{ panel.title }}
```
Comment (broslukasz) [inline src/app/statement/statement.component.html:12]: I think a translation should be don here

File: src/app/statement/statement.component.ts (hunk 2 above)
Comment (broslukasz) [reply src/app/statement/statement.component.ts:33]: Looks like it would be better to pass here keys only

**Review by bbartosiak — APPROVED** [review]

**Pull request merged**